scripts/fetch-api.sh

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env bash
+set -euxo pipefail
+
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+endpoints=(
+    "https://raw.githubusercontent.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/main/openapi/kas-fleet-manager.yaml"
+    "https://raw.githubusercontent.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/main/openapi/connector_mgmt.yaml"
+    "https://raw.githubusercontent.com/bf2fc6cc711aee1a0c2a/srs-fleet-manager/main/core/src/main/resources/srs-fleet-manager.json"
+    "https://raw.githubusercontent.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/main/openapi/smartevents_mgmt_v2.yaml"
+    "https://raw.githubusercontent.com/bf2fc6cc711aee1a0c2a/kafka-admin-api/main/kafka-admin/.openapi/kafka-admin-rest.yaml"
+    "https://raw.githubusercontent.com/Apicurio/apicurio-registry/main/common/src/main/resources/META-INF/openapi.json"
+    "https://sso.redhat.com/auth/realms/redhat-external/apis/openapi.yaml"
+)
+
+for endpoint in ${endpoints[*]}; do
+    filename=$(echo "${endpoint##*/}")
+
+    if [[ ${filename} == "openapi.yaml" ]]; then
+        filename="service-accounts.yaml"
+    fi
+    if [[ ${filename} == "openapi.json" ]]; then
```

Comment:
The name should probably be `openapi-gen.json` like so:
```
https://raw.githubusercontent.com/Apicurio/apicurio-registry/main/app/src/main/resources-unfiltered/META-INF/resources/api-specifications/registry/v2/openapi-gen.json
```

Reply:
I have used a different file:
`https://raw.githubusercontent.com/Apicurio/apicurio-registry/main/common/src/main/resources/META-INF/openapi.json`

this is the source of truth for the Server Side codegen, any specific reason for using the alternative one?

Reply:
The `*-gen` is specifically modified for the Openapi generator to generaate stuff we need in the client.

Reply:
Agreed, done here: https://github.com/redhat-developer/app-services-sdk-core/pull/98/commits/2c578b5bc193c8d9668fc234c52337424b184f3b 👍